packages/flutter_tools/test/general.shard/ios/fallback_discovery_test.dart

```diff
@@ -104,7 +104,7 @@ void main() {
       ];
     });
     when(mockVmService.getIsolate(any)).thenAnswer((Invocation invocation) async {
-      return Sentinel(kind: 'Something', valueAsString: 'Something');
+      return Sentinel(kind: 'Something', valueAsString: 'Something') as Isolate;
```

Comment:
Need to figure out the right thing to do here

Reply:
Sentinels are thrown now, according to https://pub.dev/packages/vm_service#changelog